### DISCUSSION TITLE
Question - why are additional users trial accounts

### DISCUSSION THREAD

**Discussion by OlayaGonzalez** [plausible/analytics#2509] [Anything Else]
Hi there!

I have added multiple additional users as Admins to our account. Whilst they can login independently and are able to see all stats, they appear to be in a trial.

![image](https://user-images.githubusercontent.com/31039447/206200678-f74adbc3-77d4-4427-bc23-08a76d98bf8e.png)

What does this actually mean? And why are those accounts not showing up the same Setting details as the 'main' account? For example, the way it works now, our finance officer wouldn't be able to have their own separate login, as they wouldn't be able to access and download invoices. 

Thank you!

**Comment by metmarkosaric** [comment]
thanks for the feedback @OlayaGonzalez! 

the trial note can be ignored. viewers and admins don't need to have a paid subscription and can access the stats even without a subscription. even after their trial period expires, they'll keep having access to the stats without issues. the trial period is optional and it does allow them to add their own sites if they'd like that and create their own subscriptions.

at the moment only the owner of the subscription can access invoices in their Plausible account. the invoices are also sent to the email address with which the subscription was created within Paddle (our payment processor). we plan to improve this in the future and build something like team accounts. you can read more here: https://github.com/plausible/analytics/discussions/1350
